source/index.html.md

```diff
@@ -540,26 +554,119 @@ Exactly one payment method must be used.
   <dd>
     Amount in minor units of given currency (e.g. cents if in Euro).
   </dd>
+
   <dt>currency
     <span class="type">[A-Z]{3}</span>
   </dt>
   <dd>
     <a target="_blank" href="currencies.txt">3-letter currency code</a>. (Some exponents differ from ISO 4217.)
   </dd>
+
+  <dt>credential
+    <span class="type">(<code>storing</code>|<code>using</code>)</span>
+  </dt>
+  <dd>
+    Indicate if the payment credential (e.g. PAN and expiry) will be stored for
+    future use where the payment credential is not provided by the
+    cardholder but collected from (encrypted) storage.
+    <br />
+    <code>storing</code>: The payment credential will be stored; may only be
+    stored if the authorization is approved.
+    <br />
+    <code>using</code>: The payment credential has already been stored and is now
+    being used.
+    <br />
+    Default:
+    <ul>
+      <li><code>using</code>, if <code>initiator</code> is <code>merchant</code>
+      (<code>storing</code> is invalid),</li>
+      <li><code>storing</code>, if the authorization is first-in-series.</li>
+    </ul>
```

Comment:
Consequence of https://github.com/clearhaus/gateway-api-docs/pull/111/files#r524296900.

```suggestion
    <code>store</code>: The payment credential will be stored; may only be
    stored if the authorization is approved.
    <br />
    <code>use</code>: The payment credential has already been stored and is now
    being used.
    <br />
    Default:
    <ul>
      <li><code>use</code>, if <code>initiator</code> is <code>merchant</code>
      (<code>store</code> is invalid),</li>
      <li><code>store</code>, if the authorization is first-in-series.</li>
    </ul>
```